docs/pages/versions/unversioned/sdk/image.mdx

```diff
@@ -31,7 +31,7 @@ import { YesIcon, NoIcon, PendingIcon } from '~/ui/components/DocIcons';
 | :--------: | :---------: | :---------: | :-----------------------------------------------------------------: |
 |    WebP    | <YesIcon /> | <YesIcon /> |        <YesIcon /> [~96% adoption](https://caniuse.com/webp)        |
 | PNG / APNG | <YesIcon /> | <YesIcon /> | <YesIcon /> / <YesIcon /> [~96% adoption](https://caniuse.com/apng) |
-|    AVIF    | <YesIcon /> | <YesIcon /> |      <PendingIcon /> [~79% adoption](https://caniuse.com/avif)      |
+|    AVIF    | <YesIcon /> (animated AVIFs not supported yet)| <YesIcon /> |      <PendingIcon /> [~79% adoption](https://caniuse.com/avif)      |
```

Comment:
Hey @RRaideRR, thanks for suggesting these changes. Can you backport these changes to SDK 48 doc too?

Reply:
thanks for your suggestion. I've added that also in the SDK 48 documentation.